rllib_contrib/a3c/pyproject.toml

```diff
@@ -12,7 +12,7 @@ version = "0.1.0"
 description = ""
 readme = "README.md"
 requires-python = ">=3.7, <3.11"
-dependencies = ["gym[accept-rom-license]", "gymnasium[mujoco]==0.26.3", "higher", "ray[rllib]==2.3.1"]
+dependencies = ["gym[accept-rom-license]", "gymnasium[mujoco]==0.26.3", "ray[rllib]==2.3.1"]
```

Comment:
According to our requirements, higher is used for MAML on PyTorch. Not sure why it went here in the first place.
I have no idea what the library does, just wanna point out this irregularity.

Reply:
It slipped through